ffmuc-mesh-vpn-wireguard-vxlan/luasrc/lib/gluon/gluon-mesh-wireguard-vxlan/get-user-agent-infos.lua

```diff
@@ -0,0 +1,21 @@
+local json = require 'jsonc'
+local util = require 'gluon.util'
+local info = require 'gluon.info'
+
+-- Get System Infos
+local data = assert(json.parse(util.exec('ubus call system board')), "Malformed JSON response, wrong JSON format")
+
+assert(data.release, "Malformed JSON response, missing required value: release")
+local Openwrt_version = assert(data.release.version, "Malformed JSON response, missing required value: version")
```

Comment:
I would prefer consistent naming 
```suggestion
local openwrt_version = assert(data.release.version, "Malformed JSON response, missing required value: version")
```
(Note: must be changed in L21 too)

Reply:
fixed with https://github.com/freifunk-gluon/community-packages/pull/105/commits/15dfb38139683769f352a576e12b8bf57d06839e